src/XamlX.IL.Cecil/Comparers/CecilTypeComparisonMode.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace XamlX.TypeSystem;
+
+public enum CecilTypeComparisonMode {
+    Exact,
```

Comment:
Exact is used in type caching, as it's a default value. We probably could go away with SignatureOnly for this purpose, but I didn't try.